src/terrain/tile.cpp

```diff
@@ -9,88 +9,93 @@
 namespace terrain {
 
 Tile::Tile() {
-    init({1, 1, 1}, false);
+    init({1, 1, 1}, 0);
```

Comment:
NO

0 IS NOT NULL. NULL IS IMPLEMENTATION DEFINED. THIS IS UB

USE [`nullptr`](https://en.cppreference.com/w/cpp/language/nullptr) THIS IS C++

Reply:
Did you change anything?

Reply:
well it's nolong a pointer so...